src/pages/Admin.js

```diff
@@ -34,37 +37,340 @@ const RiseUp = styled.div`
   animation: ${riseUp} 0.75s ease-in-out;
 `;
 
-const ContentWrapper = styled.div`
+const ContentWrapper = styled(Card)`
   display: flex;
-  flex-direction: row;
-  justify-content: space-around;
+  flex-direction: column;
+  align-items: center;
+  padding: 80px 100px;
 `;
 
-const RightPanels = styled(Card)`
+const SectionWrapper = styled.div`
   display: flex;
-  flex-direction: column;
-  width: 400px;
+  flex-direction: row;
+  padding-bottom: 40px;
 `;
 
-const ProposalPanel = styled(Card)`
-  max-width: 650px;
-  padding: 0px 25px 18px 25px;
+const SectionTitle = styled.p`
+  font-size: 22px;
+  color: ${theme.text.darker_default};
+  line-height: normal;
+  font-weight: 500;
+`;
+
+const SectionText = styled.p`
+  text-align: left;
+  line-height: 30px;
+  margin-top: 5px;
+  font-size: 17px;
   color: #546978;
+  margin-bottom: 20px;
+`;
+
+const StyledBody = styled.p`
+  width: 150px;
+  text-align: left;
+  line-height: 30px;
+  margin-top: 5px;
+  font-size: 17px;
+  color: #546978;
+`;
+
+const OptionText = styled.p`
+  text-align: left;
   line-height: 30px;
+  font-size: 17px;
+  color: #546978;
+`;
+
+const VoteOptionsGrid = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-column-gap: 10px;
+  grid-row-gap: 10px;
+`;
+
+const CloseIcon = styled.p`
+  width: 15px;
+  height: 15px;
+  background-color: red;
+  mask: url(${closeImg}) center no-repeat;
+  cursor: pointer;
+`;
+
+const Code = styled.pre`
+  font-size: 14px;
+  padding: 30px;
+  border: 1px solid black;
 `;
 
+const ABSTAIN = 'Abstain';
+const NOCHANGE = 'No Change';
+
 class Admin extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      pollTitle: '',
+      pollSummary: '',
+      pollStart: null,
+      pollEnd: null,
+      pollLink: '',
+      pollRules: '',
+      pollOption: '',
+      pollOptions: [ABSTAIN, NOCHANGE],
+      pollContent: '',
+      pollMarkdown: '',
+      pollGenerated: false
```

Comment:
it's a stylistic choice, and neither way is wrong, but b/c each of these variables begin with `poll`, we could store them all in an object like `poll: { title: '', summary: '', ... }`. Then, we can use that namespace to provide context when we go to use them. For example, `value={poll.link}` rather than `value={pollLink}`